src/hb-ot-var.cc

```diff
@@ -77,9 +82,17 @@ hb_ot_var_get_axis_count (hb_face_t *face)
 
 /**
  * hb_ot_var_get_axes:
+ * @face: #hb_face_t to work upon
+ * @start_offset: offset of the first lookup to retrieve
+ * @axes_count: (inout) (allow-none): Input = the maximum number of variation axes to return;
+ *                Output = the actual number of variation axes returned (may be zero)
+ * @axes_array: (out) (array length=axes_count): The array of variation axes found
+ *
+ * Fetches a list of all variation axes in the specified face. The list returned will begin
+ * at the offset provided.
  *
  * Since: 1.4.2
- * Deprecated: 2.2.0
+ * Deprecated: 2.2.0 - use #hb_ot_var_get_axis_infos instead
```

Comment:
```suggestion
 * Deprecated: 2.2.0: use hb_ot_var_get_axis_infos() instead
```

Based on the example from https://developer.gnome.org/gtk-doc-manual/stable/documenting_symbols.html.en